Float.TinCan.ActivityLibrary/ActivityLaunchCoordinator.cs

```diff
@@ -127,8 +128,7 @@ public override void Start()
 
                     downloadStatus.DownloadsCompleted += HandleDownloadCompleted;
                     downloadStatus.DownloadsCancelled += HandleDownloadCancelled;
-
-                    NavigationContext.PresentPage(CreateDownloadStatusPage(downloadStatus));
+                    await ShowDownloadStatus(CreateDownloadStatusPage(downloadStatus));
```

Comment:
@EBusch Could we get into a state where we are creating/presenting the download page more than once?
_just wondering if we need a protection from creating multiple_